app/views/members/_search.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= form_tag(members_path, method: :get) do %>
-  <%= text_field_tag :query, params[:query] %>
+  <%= text_field_tag "member[query]", sanitize(params[:query]) %>
```

Comment:
This change to `"member[query]"` doesn't match the value being reflected back with `params[:query]`. With this change, that would need to become:

```ruby
sanitize(params[:member][:query])
```

Now might be a good time for a helper method :)